### DISCUSSION TITLE
Git commits don't show up as contribution - issue with email

### DISCUSSION THREAD

**Discussion by talostudios** [community/community#35393] [Other Feature Feedback, Questions, & Ideas]
My account was previously associated with oldEmail@example.com and is now associated with my currentEmail@example.com. If I go to Settings > Email, I can see that currentEmail@example.com is verified and set to primary (as advised by this [doc](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account)).

I'm not seeing my contributions on the overview. If I go to a recent commit and add .patch to the end of the URL, I can see that it's still associated with oldEmail@example.com, so I know that's the problem. However, in this scenario, we're supposed to add our new email ([doc](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account)) which I've done, so not sure why the issue is persisting. Any thoughts?

I found similar discussions, but their issues were resolved once they added their current email.

EDIT:

One thing I noticed is that the [add email doc](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account) step 5 shows a screenshot that says the primary email will be used for edits and merges. 
<img width="729" alt="image" src="https://user-images.githubusercontent.com/23443760/194471747-d5e2317d-9ee3-4aed-b6a5-de03b3339457.png">

However, on my settings it only says: email will be used for account-related notifications and can be used for password resets. Why the discrepancy?
<img width="620" alt="image" src="https://user-images.githubusercontent.com/23443760/194471921-69a43352-835c-4014-98c0-d34ef5d4c321.png">

**Comment by manjeetk** [comment]
EDIT - Apologies, this is not the suggested answer but I am having the same issue.

Hi,
I also have the same issue. Contributions from old email address are not showing on the new email address. I have also added old email address to the new email address github account.

**Reply by manjeetk** [reply]
Could anyone please advise how to fix this? Thanks

**Comment by manjeetk** [comment]
Hi @talostudios 
Hope you are well, Just wondering if you were able to resolve this issue yet?

**Reply by talostudios** [reply]
Hi @manjeetk -- Unfortunately, no. I'm going to try and troubleshoot myself a bit more.
